src/Handpose/index.js

```diff
@@ -61,19 +62,42 @@ class Handpose {
   async loadModel() {
     const pipeline = handPoseDetection.SupportedModels.MediaPipeHands;
     //filter out model config options
-    const modelConfig = {
-      maxHands: this.config?.maxHands ?? 2,
-      runtime: this.config?.runtime ?? "mediapipe",
-      modelType: this.config?.modelType ?? "full",
-      solutionPath:
-        this.config?.solutionPath ??
-        "https://cdn.jsdelivr.net/npm/@mediapipe/hands",
-      detectorModelUrl: this.config?.detectorModelUrl,
-      landmarkModelUrl: this.config?.landmarkModelUrl,
-    };
-    this.runtimeConfig = {
-      flipHorizontal: this.config?.flipHorizontal ?? false,
-    };
+    const modelConfig = handleOptions(this.config, {
+      maxHands: {
+        type: "number",
+        min: 1,
+        default: 2,
+      },
+      runtime: {
+        type: "enum",
+        enums: ["mediapipe", "tfjs"],
+        default: "mediapipe",
+      },
+      modelType: {
+        type: "enum",
+        enums: ["lite", "full"],
+        default: "full",
+      },
+      solutionPath: {
+        type: (config) =>
+          config.runtime === "mediapipe" ? "string" : "undefined",
+        default: "https://cdn.jsdelivr.net/npm/@mediapipe/hands",
+      },
+      detectorModelUrl: {
+        type: (config) => (config.runtime === "tfjs" ? "string" : "undefined"),
+        default: undefined,
+      },
+      landmarkModelUrl: {
+        type: (config) => (config.runtime === "tfjs" ? "string" : "undefined"),
```

Comment:
If this sort of ternary is showing up everywhere then potentially you could have an additional property to handle that situation.  Like 
```
detectorModelUrl: {
  type: "string",
  default: undefined,
  disallowed: (config) => config.runtime !== "tfjs"
},
```

Reply:
I think this is a good idea!